pages/team/[id].tsx

```diff
@@ -0,0 +1,318 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
```suggestion
```

I removed some linting rules so it's complaining that it doesn't know what this rule is. You can just remove it and keep the `@ts-nocheck` for now. I'll fix the type errors later 😄

Reply:
I fixed 'em! Guess I can't ignore TS errors anymore 👀